src/Resources/views/standard_layout.html.twig

```diff
@@ -159,22 +159,26 @@ file that was distributed with this source code.
                                 <div class="navbar-custom-menu">
                                     <ul class="nav navbar-nav">
                                         {% block sonata_top_nav_menu_add_block %}
-                                            <li class="dropdown">
-                                                <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-                                                    <i class="fas fa-plus-square fa-fw" aria-hidden="true"></i> <i class="fas fa-caret-down" aria-hidden="true"></i>
-                                                </a>
-                                                {% include get_global_template('add_block') %}
-                                            </li>
+                                            {% if include(get_global_template('add_block')) is not empty %}
+                                                <li class="dropdown">
+                                                    <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+                                                        <i class="fas fa-plus-square fa-fw" aria-hidden="true"></i> <i class="fas fa-caret-down" aria-hidden="true"></i>
+                                                    </a>
+                                                    {% include get_global_template('add_block') %}
```

Comment:
to avoid double include you could save the initial include to a variable.

Reply:
Then I need to use the raw filter since the variable will have html, do you prefer this way ?

Reply:
I feel like this is better since we don't load 2 times the templates, but wdyt?

Reply:
I don't really have an opinion about it, let's go that way.